plugins/InspectionGadgets/testsrc/com/siyeh/ig/naming/OverloadedVarargsMethodInspectionTest.java

```diff
@@ -64,4 +64,29 @@ public void testGenericMethods() {
            "        }" +
            "    }");
   }
+
+  public void testNoWarningBecauseOfTypes() {
+    doTest("class Overload {" +
+           "  public void method() {}" +
+           "  public void method(int p1) {}" +
+           "  public void method(int p1, int p2) {}" +
+           "  public void method(int p1, String p2, int p3) {}" +
+           "  public void method(int p1, String p2, String p3, int p4) {}" +
+           "  public void method(int p1, String p2, String... p3) {}" +
+           "}");
+  }
+
+  public void testWarningBecauseOfInheritance() {
```

Comment:
I see no inheritance in the test case? Maybe rename?

Reply:
Thank you. Renamed